### PR TITLE
adding code in dev1.0 branch

### PR DESCRIPTION
**Pull request by sudheerkumarazure1992**
Hi @chandra-azure ,

Can you review terraform code.

### CODE REVIEW

File: terraform.txt [hunk 1]
```diff
@@ -0,0 +1,16 @@
+module "ec2_instance" {
+  source  = "terraform-aws-modules/ec2-instance/aws"
+
+  name = "single-instance"
+
+  instance_type          = "t2.micro"
```
Comment (chandra-azure) [inline terraform.txt:6]: please change instance_type as t.large

File: terraform.txt [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  key_name               = "user1"
+  monitoring             = true
+  vpc_security_group_ids = ["sg-12345678"]
+  subnet_id              = "subnet-eddcdzz4"
+
+  tags = {
+    Terraform   = "true"
+    Environment = "dev"
```
Comment (chandra-azure) [inline terraform.txt:14]: change prod

**Review by chandra-azure — COMMENTED** [review]
can you work on review comments

File: terraform.txt (hunk 1 above)
Comment (sudheerkumarazure1992) [reply terraform.txt:6]: added new instance type

**Comment by sudheerkumarazure1992** [comment]
added code

**Review by chandra-azure — APPROVED** [review]
looks good

**Pull request merged**